src/app/form/(formSteps)/FormLayout.tsx

```diff
@@ -1,9 +1,66 @@
+import { allSections, getCurrentFormProgress } from "@form/_utils/formProgress";
+import { RequiredMarker } from "@trussworks/react-uswds";
+
+type CompletionState = "complete" | "current" | "incomplete";
+
 // Separated this into a separate testable component because as of writing, Jest does not support testing NextJs asynchronous server components (https://nextjs.org/docs/app/guides/testing/jest)
 export const FormLayout = (props: { children?: React.ReactNode; pathname: string }) => {
+  const { section: currentSection } = getCurrentFormProgress(props.pathname);
+  const currentSectionIndex = allSections.findIndex(
+    (sections) => sections.id === currentSection.id,
+  );
+
   return (
-    <div>
-      <h1 className="margin-top-4">{props.pathname}</h1>
-      <div className="margin-top-4">{props.children}</div>
-    </div>
+    <>
+      <div className="usa-step-indicator" aria-label="progress">
```

Comment:
[pebble] I don't like "progress" here. Can I suggest something like:
```suggestion
      <div className="usa-step-indicator" aria-label="Form progress">
```